changelogs/fragments/473-nmcli-slave-type-implemented.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - nmcli - new module option 'slave_type' added to allow creation of various types of slave devices (https://github.com/ansible-collections/community.general/issues/473).
```

Comment:
```suggestion
  - nmcli - new module option ``slave_type`` added to allow creation of various types of slave devices (https://github.com/ansible-collections/community.general/issues/473).
```

Reply:
Fixed in [858b601](https://github.com/ansible-collections/community.general/pull/6108/commits/858b6011503d524aef7016d6640ee4b2627c70cd)